pkgs/apps/polyply/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, buildPythonPackage, fetchFromGitHub, numpy, scipy, vermouth, pbr
+, tqdm, numba, pytestCheckHook }:
+
+buildPythonPackage rec {
+  pname = "polyply";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "marrink-lab";
+    repo = "${pname}_1.0";
```

```suggestion
    repo = "polyply_1.0";
```
There's not real benefit of using pname here in my opinion. I think it is more readable just spelling the name out.